index.html

```diff
@@ -128,112 +154,119 @@ <h1 class="cardonept">
-                <img src="img/Snapshoot Portfolio-5.png" alt="">
+                <div class="imagefit">
+                    <img class="mobile" src="img/Snapshoot Portfolio-5.png" alt="">
+                    <img class="desktop" src="img/Snapshoot Portfolio 3.png" alt="">
+
+                </div>
                 <div>
                     <h1 class="cardonept">
                         Multi-Post
                         Stories
                     </h1>
-                    <ul class="frame2">
-                        <li id="listid">CANOPY</li>
-                        <!-- <img src="img/Counter.png" alt=""> -->
-                        <li class="lineitem">Back End Dev</li>
-                        <!-- <img src="img/Counter.png" alt=""> -->
-                        <li class="lineitem">2015</li>
-                    </ul>
-
-                    <p class="tagtext">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-
-                    <ul class="frame2">
-                        <li class="tag">html </li>
-                        <li class="tag">css</li>
-                        <li class="tag">javaScript</li>
-                    </ul>
-
-                    <div>
-                        <div class="action">
-                            <button class="buttonaction">See Project</button>
+                    <div class="pconeup">
+                        <ul class="frame2">
+                            <li id="listid">CANOPY</li>
+                            <!-- <img src="img/Counter.png" alt=""> -->
+                            <li class="lineitem">Back End Dev</li>
+                            <!-- <img src="img/Counter.png" alt=""> -->
+                            <li class="lineitem">2015</li>
+                        </ul>
+
+                        <p class="tagtext">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+
+                        <ul class="frame2">
+                            <li class="tag">html </li>
+                            <li class="tag">css</li>
+                            <li class="tag">javaScript</li>
+                        </ul>
+
+                        <div>
+                            <div class="action">
+                                <button class="buttonaction">See Project</button>
+                            </div>
                         </div>
                     </div>
-
                 </div>
 
 
 
             </section>
 
-            <section class="cardone">
-                <img src="img/Snapshoot Portfolio-2.png" alt="">
+            <section class="cardone inverse">
+                <div class="imagefit">
+                    <img class="mobile" src="img/Snapshoot Portfolio-2.png" alt="">
+                    <img class="desktop" src="img/Snapshoot Portfolio 4.png" alt="">
```

Comment:
[OPTIONAL] 👍🏽 

- If you have the time, I'd like to kindly suggest that you increase the width and height of the images in your last two project cards. They are a bit too small. 

Here is what I mean 👇🏽 


| Your Project card image  |
| :---:   | 
|![small](https://user-images.githubusercontent.com/99246359/208233952-496de623-b449-473b-b42f-29c479363e73.PNG)  |